### PR TITLE
APIGOV-28401 SoftwareAG WebMethods documentation

### PR DESCRIPTION
**Pull request by alrosca**
SoftwareAG WebMethods documentation

http://pr-516.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_software_ag_webmethods/

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 6986cc2 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-516.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/connect_manage_environ/connect_software_ag_webmethods/agent-variables.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+| WEBMETHODS_AUTH_USERNAME             | The username used to authenticate the calls to SoftwareAG WebMethods gateway (same as the one used to login)                                                                    |
+| WEBMETHODS_AUTH_PASSWORD             | The password used to authenticate the calls to SoftwareAG WebMethods gateway (same as the one used to login)                                                                    |
+| WEBMETHODS_TIMEOUT                   | The connection timeout for querying SoftwareAG WebMethods's API (default: `1m`)                                                                                                 |
+| WEBMETHODS_POLLINTERVAL              | The interval in which the agent will query SoftwareAG WebMethods for new APIs or transactions. (Discovery default: `5m`, `1m` minimum, Traceability default and minimum: `15m`) |
+| WEBMETHODS_OAUTH2AUTHZSERVERALIAS    | The OAuth2 server alias used by SoftwareAG WebMethods to authenticate with OAuth2 creedntial                                                                                    |
```
Comment (jcollins-axway) [inline content/en/docs/connect_manage_environ/connect_software_ag_webmethods/agent-variables.md:34]: creedntial

File: content/en/docs/connect_manage_environ/connect_software_ag_webmethods/agent-variables.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+| WEBMETHODS_URL                       | The base URL for connect to SoftwareAG WebMethods.                                                                                                                              |
+| WEBMETHODS_AUTH_USERNAME             | The username used to authenticate the calls to SoftwareAG WebMethods gateway (same as the one used to login)                                                                    |
+| WEBMETHODS_AUTH_PASSWORD             | The password used to authenticate the calls to SoftwareAG WebMethods gateway (same as the one used to login)                                                                    |
+| WEBMETHODS_TIMEOUT                   | The connection timeout for querying SoftwareAG WebMethods's API (default: `1m`)                                                                                                 |
+| WEBMETHODS_POLLINTERVAL              | The interval in which the agent will query SoftwareAG WebMethods for new APIs or transactions. (Discovery default: `5m`, `1m` minimum, Traceability default and minimum: `15m`) |
```
Comment (jcollins-axway) [inline content/en/docs/connect_manage_environ/connect_software_ag_webmethods/agent-variables.md:33]: This is great, but could we update it (and the agent code) to default TA to 1 hour, minimum of 15m to match how the SDK handles usage reporting?

**Review by jcollins-axway — APPROVED** [review]

**Review by robertengaxway — APPROVED** [review]
Approved

**Pull request merged**